clustered/client/src/main/java/org/ehcache/clustered/client/config/TimeoutDuration.java

```diff
@@ -0,0 +1,159 @@
+/*
+ * Copyright Terracotta, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.ehcache.clustered.client.config;
+
+import java.math.BigInteger;
+import java.util.concurrent.TimeUnit;
+
+/**
+ * Describes a timeout value for a clustered operation.  Instances of this class
+ * are subject to the operation of the Java {@code java.util.concurrent.TimeUnit} class.
+ */
+public final class TimeoutDuration {
```

Comment:
As discussed -- I'll do another PR to move `TimeoutDuration` into the API module under another name to be chosen -- maybe `TimeSpan` -- along with a new method in `org.ehcache.xml.Conversions` (or maybe another utility class like `Parser`) to support JAXB parsing of one from the JAXB `TimeUnit`.